app/pods/components/utils/documents-list-for-item/template.hbs

```diff
@@ -4,7 +4,7 @@
   <div class="grid-container">
     {{#each (await documents) as |document|}}
       {{item-document
-        item=@item
+        item=@agendaitemOrSubcase
```

Comment:
item in dit geval (en voor document-list-for-item) is momenteel agendaitemOrSubcase.
maar is generiek genoeg om te werken voor elk model met de relatie hasMany('document-version')
Dus de vraag is een beetje hoe we dit soort generieke variabelen willen noemen.
Ik zou item van item-document voorlopig laten staan, maar zet er een TODO bij.
We willen gaan naar een situatie waar document-list-for-item en item-document zo generiek zijn, dat ze enkel een lijst van documenten (document-versions) binnenkrijgen. Dan maakt het model niet meer uit en kunnen we deze variabele gewoon `document` noemen.

Reply:
Ok, ik laat deze momenteel.